README.md

```diff
@@ -5,8 +5,8 @@ TextAnalysis
 A Julia package for working with text.
 
 [![Build Status](https://api.travis-ci.org/JuliaText/TextAnalysis.jl.svg)](https://travis-ci.org/JuliaText/TextAnalysis.jl)
-[![TextAnalysis](http://pkg.julialang.org/badges/TextAnalysis_0.5.svg)](http://pkg.julialang.org/?pkg=TextAnalysis)
 [![TextAnalysis](http://pkg.julialang.org/badges/TextAnalysis_0.6.svg)](http://pkg.julialang.org/?pkg=TextAnalysis)
+[![TextAnalysis](http://pkg.julialang.org/badges/TextAnalysis_0.7.svg)](http://pkg.julialang.org/?pkg=TextAnalysis)
```

Comment:
unfortunately, `pkg.julialang.org` does not provide badges for 0.7 onwards. Might as well remove these altogether.

Reply:
Okay, thanks, didn't know that. Should I replace it with travis-ci and appveyor bages?

Reply:
Yeah.